src/client.ts

```diff
@@ -503,6 +512,44 @@ export class Client extends EventEmitter {
     }
   }
 
+  /**
+   * It works by subscribing to particular events. The node will return a subscription id.
+   * For each event that matches the subscription a notification with relevant data is send
+   * together with the subscription id.
+   *
+   * Possible methods:
+   *  * "NewHeads": Fires a notification each time a new header is appended to the chain
+   *  * "Logs": Returns logs that are included in new imported blocks and match the given filter criteria
+   *
+   * Example of a "filter" (JSON String) with method "logs":
+   *  {
+   *    "address": "0xa520fe7702b96808f7bbc0d4a233ed1468216cfd",
+   *    "topics": ["0x238a0cb8bb633d06981248b822e7bd33c2a35a6089241d099fa519e361cab902"]
+   *  }
+   *
+   * @param method Method selected to the filter, can be "newHeads" or "logs"
+   * @param filter JSON string of the filter
+   */
+  async evmSubscribeAsync(method: string, filter: string): Promise<string> {
```

Comment:
I think `filter` should be passed in as an `object`, and stringified in this function.

Reply:
yep, that's make sense :)